### ISSUE TITLE
Suggestion for iOS users and Browser 

### ISSUE THREAD

**Issue by Chippel** [privacytools/privacytools.io#1989]
So why not recommending Safari with an Adblock like AdGuard which is open source and blocks trackers and ads on the browser? Why is there DuckDuckGo in the list as app?

**Comment by FreddyMarsden** [comment]
DuckDuckGo is listed under [Browser Recommendations For iOS](https://www.privacytools.io/browsers/#browser-ios). Safari is closed source, and while their new features are a step in the right direction, they aren't worth recommending. Using Firefox with all the [add-ons](https://www.privacytools.io/browsers/#addons) and [tweaks](https://www.privacytools.io/browsers/#about_config) along with using [encrypted DNS](https://www.privacytools.io/providers/dns/) will rule out any need for AdGaurd.

**Comment by ThracianKnight1907** [comment]
Firefox on iOS is required to use the Safari engine to render pages (as the rest of the browsers on iOS). There is no possibility to install add-ons and tweaking it is severly limited. The PrivacyTools page you linked is only for desktop and Android (at least some of them work on Android). 

Safari on iOS doesn't support normal adblockers so it uses content blockers. AdGuard+Safari isn't as powerful as uBlockOrigin+Firefox (on Android/Linux/Windows/Mac etc.), but is better from my experience than Firefox iOS or DDG browser.

**Comment by FreddyMarsden** [comment]
@ThracianKnight1907 for Firefox I was referring to desktop (which is what OP seemed to be talking about).

**Comment by ookangzheng** [comment]
If you dont mind, better to use iOS 13, at least they block more ads, tracker

**Comment by jneplokh** [comment]
Yeah I feel it is better we recommend Safari with something like AdGuard.

Safari already has [fingerprinting protections by default](https://www.apple.com/safari/docs/Safari_White_Paper_Nov_2019.pdf) and will allow a user to blend in more with the vast majority of iPhones. If a user is worried about cookies, they could just always be in Private mode.

**Comment by github-account1111** [comment]
Actually I think the iOS section should *only* contain Safari and have the browser recs that are currently there removed.

Just think logically: Apple already has your info. Would you rather also give that info to a second company or keep it limited to Apple? I'd rather keep it limited to Apple.

Not to mention Safari already has some fingerprint and tracking protection in place, so it's not a bad choice at all even from that perspective.

**Comment by jneplokh** [comment]
I completely agree. Not only does it reduce the amount of trusted parties as you stated, but those fingerprinting and tracking protections are honestly very good, especially in comparison to other browsers.
